transcoder/ffmpeg_segment_failcase_test.go

```diff
@@ -0,0 +1,266 @@
+// +build nvidia
+
+package transcoder
+
+import (
+	"bufio"
+	b64 "encoding/base64"
+	"encoding/csv"
+	"encoding/json"
+	"fmt"
+	"github.com/livepeer/lpms/ffmpeg"
+	"io"
+	"io/ioutil"
+	"os"
+	"path"
+	"path/filepath"
+	"strconv"
+	"strings"
+	"testing"
+	"time"
+)
+
+func selectFile(files *[]string) filepath.WalkFunc {
+	return func(path string, info os.FileInfo, err error) error {
+		if err != nil {
+			return err
+		}
+		if info.IsDir() || filepath.Ext(path) != ".txt" {
+			return nil
+		}
+		*files = append(*files, path)
+		return nil
+	}
+}
+func getTsFilename(txtpath string) string {
+	txtfname := path.Base(txtpath)
+	tsfname := ""
+	i := strings.Index(txtfname, ".ts-")
+	if i > 0 {
+		tsfname = txtfname[:i] + ".ts"
+	}
+	return tsfname
+}
+func parseProfiles(instr string) []ffmpeg.VideoProfile {
```

Comment:
Given this is used at multiple places (livepeer_bench another example), maybe we should move this some place common, like `ffmpeg/videoprofile.go`, to avoid duplication?

Reply:
Good Idea, then we also need to move the following functions.

DefaultProfileName,
EncoderProfileNameToValue

And last one thing,  we have to refactor go-livepeer part related to these functions. WDYT?

Reply:
>  then we also need to move the following functions.
DefaultProfileName,
EncoderProfileNameToValue
And last one thing, we have to refactor go-livepeer part related to these functions. 

Yeah makes sense on moving these functions from `go-livepeer/common` to `lpms/ffmpeg` as well. Feel free to do this refactoring separately from this PR if you think that is cleaner.